### ISSUE TITLE
Security vulnerabilities

### ISSUE THREAD

**Issue by jrhorn424** [dawnlabs/alchemy#47]
The following output was produced on Ubuntu 18.04.

```
jeffh@ubuntu-vbox:~/projects/alchemy$ npm install
npm WARN deprecated github@9.2.0: 'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)
npm WARN deprecated chalk@2.0.1: Please upgrade to Chalk 2.1.0 - template literals in this version (2.0.1) are quite buggy.
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated native-or-bluebird@1.2.0: 'native-or-bluebird' is deprecated. Please use 'any-promise' instead.
npm WARN deprecated hoek@2.16.3: The major version is no longer supported. Please update to 4.x or newer

> node-sass@4.9.4 install /home/jeffh/projects/alchemy/node_modules/node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.9.4/linux-x64-64_binding.node
Download complete .] - :
Binary saved to /home/jeffh/projects/alchemy/node_modules/node-sass/vendor/linux-x64-64/binding.node
Caching binary to /home/jeffh/.npm/node-sass/4.9.4/linux-x64-64_binding.node

> electron@1.8.8 postinstall /home/jeffh/projects/alchemy/node_modules/electron
> node install.js

Downloading SHASUMS256.txt
[============================================>] 100.0% of 5.74 kB (5.74 kB/s)

> node-sass@4.9.4 postinstall /home/jeffh/projects/alchemy/node_modules/node-sass
> node scripts/build.js

Binary found at /home/jeffh/projects/alchemy/node_modules/node-sass/vendor/linux-x64-64/binding.node
Testing binary
Binary is fine
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1678 packages from 829 contributors and audited 30520 packages in 79.476s
found 8 vulnerabilities (1 low, 4 moderate, 2 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```

Would you accept a PR with automatic fixes applied?

**Comment by mfix22** [comment]
Absolutely! 👍

**Comment by jrhorn424** [comment]
The following vulnerabilities remain after automatic fixes. I'll send the PR with automatic fixes shortly.

```
jeffh@hook:~/projects/alchemy on (jrh-update-vulnerable-packages)! in .
% npm audit

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ string                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ string                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ string                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/536                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ open                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ release [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ release > open                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/663                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ release [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ release > git-spawned-stream > debug                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/534                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 3 vulnerabilities (1 low, 1 high, 1 critical) in 30559 scanned packages
  3 vulnerabilities require manual review. See the full report for details.
```

If you'd like me to look into these, I'm happy to.
